### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by nekongum**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/SarutaFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class SarutaFeature {
+    public void feature() {
+        // this is a new feature
+    }
```
Comment (nekongum) [inline lab-exercise-1/SarutaFeature.java:4]: This content is good

**Review by nekongum — COMMENTED** [review]
I've added my review.

File: lab-exercise-1/SarutaFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class SarutaFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/SarutaFeature.java:3]: code is empty you should fix

File: lab-exercise-1/SarutaFeature.java (hunk 1 above)
Comment (JNapat) [reply lab-exercise-1/SarutaFeature.java:6]: I think so na.

File: lab-exercise-1/SarutaFeature.java (hunk 2 above)
Comment (jaxmmmm) [inline lab-exercise-1/SarutaFeature.java:3]: It's not the best.

Comment (JNapat) [reply lab-exercise-1/SarutaFeature.java:3]: I think so.

File: lab-exercise-1/SarutaFeature.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+        int x = 0;
+        x = x + 1;
+    }
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/SarutaFeature.java:6]: I think it is good now

File: lab-exercise-1/SarutaFeature.java (hunk 2 above)
Comment (Napassawxn) [reply lab-exercise-1/SarutaFeature.java:3]: me too

File: lab-exercise-1/SarutaFeature.java (hunk 3 above)
Comment (Napassawxn) [reply lab-exercise-1/SarutaFeature.java:6]: same kubb

**Comment by rmabow03** [comment]
it not complete the feature, can you add the feature in your code.

File: lab-exercise-1/SarutaFeature.java (hunk 2 above)
Comment (rmabow03) [reply lab-exercise-1/SarutaFeature.java:3]: it not complete the feature, can you add the feature in your code.

Comment (rmabow03) [reply lab-exercise-1/SarutaFeature.java:3]: don't forget to add your feature

File: lab-exercise-1/SarutaFeature.java [hunk 4]
```diff
@@ -1,5 +1,7 @@
 public class SarutaFeature {
     public void feature() {
         // this is a new feature
+        int x = 0;
+        x = x + 1;
```
Comment (jaxmmmm) [inline lab-exercise-1/SarutaFeature.java:5]: good

File: lab-exercise-1/SarutaFeature.java (hunk 2 above)
Comment (Napassawxn) [inline lab-exercise-1/SarutaFeature.java:3]: it empty?

**Pull request merged**